pages/settings/wallets/ConnectWallet.tsx

```diff
@@ -0,0 +1,281 @@
+              router.replace("/");
+            }}
+            className="-mr-4 px-6"
+          >
+            <XIcon className="text-muted-foreground" width={24} height={24} />
+          </TouchableOpacity>
+        )}
+      />
+      <View className="flex-1 justify-center items-center gap-8 p-6">
+        <View className="flex flex-row items-center justify-center">
+          <View className="flex items-center">
+            <View className="shadow">
+              <Image
+                source={require("../../../assets/hub.png")}
+                className="my-4 rounded-2xl w-20 h-20"
+              />
+            </View>
+            <Text className="text-xl font-semibold2">Alby Hub</Text>
+          </View>
+
+          <View className="z-[-1] relative w-36 h-16 mb-4 items-center justify-center">
+            <Animated.Image
+              resizeMode="contain"
+              source={require("../../../assets/left-plug.png")}
+              className="absolute w-20 h-20"
+              style={{
+                left: -10,
+                transform: [
+                  {
+                    translateX: leftPlugAnim,
+                  },
+                ],
+              }}
+            />
+            <Animated.Image
+              resizeMode="contain"
+              source={require("../../../assets/right-plug.png")}
+              className="absolute w-20 h-20"
+              style={{
+                right: -20,
+                transform: [
+                  {
+                    translateX: rightPlugAnim,
+                  },
+                ],
+              }}
+            />
+          </View>
+
+          <View className="flex items-center">
+            <View className="shadow">
+              <Image
+                source={{ uri: appicon }}
+                className="my-4 rounded-2xl shadow-md w-20 h-20"
+              />
+            </View>
+            <Text className="text-xl font-semibold2">{appname}</Text>
+          </View>
+        </View>
+        {nwcUrl ? (
+          <View className="flex-1 w-full justify-center items-center">
+            <Tick />
+            <Text className="my-4 text-lg text-center text-foreground">
+              Connected! You're being redirected in {redirectCountdown}{" "}
+              seconds...
+            </Text>
+          </View>
+        ) : (
+          <>
+            <Text className="text-xl text-center text-foreground">
+              <Text className="text-xl font-semibold2">{appname}</Text> is
+              requesting to access your wallet{" "}
+              <Text className="text-xl font-semibold2">Alby Hub</Text> for
+              in-app payments.
+            </Text>
+            <View className="flex-1 flex mt-4 gap-8 justify-center">
+              <View>
+                {/* TODO: REPLACE WITH A PROPER COMPONENT */}
+                <TouchableOpacity className="flex flex-row border-2 border-muted justify-between items-center rounded-2xl p-6 py-4">
```

Comment:
I guess we need to be able to edit at least the budget on tap, if not explicitly set by the app

Reply:
hmm, it defaults to 100k, maybe it's ok for now as a first iteration

Reply:
Opened https://github.com/getAlby/go/issues/288 to tackle this separately